Pipfile

```diff
@@ -21,8 +21,11 @@ xmltodict = "*"
 lxml = "*"
 pytz = "*"
 tzlocal = "*"
-libvirt-python = {version = "*", sys_platform !=  "== 'win32'"}
 SQLAlchemy = "*"
+libvirt-python = {version = "*", sys_platform = "!= 'win32'"}
+flask-testing = "*"
```

Comment:
These should go to the dev-packages. We don't want flask-testing installed by default.

And mock/pipfile?

Reply:
Done. `mock` is needed for tests as well. `pipfile` was being added automatically because of my mistake of doing `pipenv install Pipfile` instead of just `pipenv install`. I wasn't aware of that. Removed.